docs/src/kubectl-plugin.md

```diff
@@ -171,6 +171,14 @@ Once the plugin was installed and deployed, you can start using it like this:
 kubectl cnpg <command> <args...>
 ```
 
+!!! Note
+    The plugin uses colorization to help make the information easier to read at
+    a glance.
+    Some users may prefer to deactivate colorization, which can be done setting
+    the `--color` flag to `never`. By default the plugin will deactivate
+    colorization when its output is sent to a non-terminal (e.g. piping to a
+    pager or `grep`).
+
```

Comment:
```suggestion
!!! Note
    The plugin automatically detects if the standard output channel is connected to a terminal.
    In such cases, it may add ANSI colors to the command output. To disable colors, use the
    `--color=always` option with the command.
```